django_tasks/backends/database/backend.py

```diff
@@ -43,6 +46,17 @@ def _task_to_db_task(
             backend_name=self.alias,
         )
 
+        # sign value without timestamp
+        # TimestampSigner doesn't support setting our own timestamp?
+        if settings.TASKS.get(self.alias, {}).get("SIGN_TASKS") is True:
```

Comment:
Issue: This is the wrong way to get the settings for the backend. Settings should be unpacked in the backend's constructor